contracts/src/base/registry/facets/distribution/v2/DelegationProxy.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.18;
+
+import {ERC20Votes} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+
+contract DelegationProxy {
```

Comment:
consider making this upgradeable by using Proxy.sol and making this into the implementation

Reply:
Done.